channels/drive/client/statvfs.c

```diff
@@ -36,10 +38,10 @@ int statvfs(const char *path, struct statvfs *buf)
 	DWORD lpTotalNumberOfClusters;
 
 	len = MultiByteToWideChar(CP_ACP, 0, path, -1, NULL, 0);
-	unicodestr = (LPWSTR) malloc(len);
+	unicodestr = (LPWSTR) malloc(len*sizeof(WCHAR));
```

Comment:
@DavBfr You can simplify the whole thing by using ```ConvertToUnicode``` from winpr. Either way you need to check for errors and handle them.